packages/noya-utils/src/__tests__/sortBy.test.ts

```diff
@@ -0,0 +1,5 @@
+import { sortBy } from '../index';
+
+test('empty array', () => {
+  expect(sortBy([], '')).toEqual([]);
```

Comment:
I'm going to merge this, but I'll create a follow-up issue to add some more tests for `sortBy`